vapoursynth/src/node/mod.rs

```diff
@@ -104,22 +105,40 @@ impl<'core> Node<'core> {
     }
 
     /// Generates a frame directly.
+    /// Returns an error if the requested frame number is beyond the `VideoInfo` num_frames.
```

Comment:
```suggestion
```
I think this goes without additional saying.

Reply:
Thanks! Fixed.